### ISSUE TITLE
Settings button messes with the GUI layout and display

### ISSUE THREAD

**Issue by yanne** [robotframework/RIDE#781]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=746) by c.martin.taylor on 14 Mar 2011*

The first 3 attached screenshots show before and after "Settings" pictures of RIDE in a maximized window. Observe how the GUI layout gets messed up.

The final 2 attached screenshots show before and after "Settings" pictures of RIDE in a smaller window.  In this scenario observe how RIDE forces a larger, off-screen, window size.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=746#c2) by @jrantanen on 24 Mar 2011*


Fixed in <a href="https://code.google.com/p/robotframework-ride/source/detail?r=84cef76c2d4a">revision 84cef76c2d4a</a>.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=746#c3) by @spooning on 28 Mar 2011*


<a class="closed_ref" href="https://code.google.com/p/robotframework-ride/issues/detail?id=748" title="Edit buttons disappeared from test case and keyword 'Settings'"> Issue 748 </a> has been merged into this issue.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=746#c4) by @pekkaklarck on 28 Mar 2011*


This issue was updated by <a href="https://code.google.com/p/robotframework-ride/source/detail?r=8c16b7fc4239">revision 8c16b7fc4239</a>.

RIDE still messed up UI after resize when settings were expanded.
We spent hours trying to debug this problem before I found "magic"
styles `wx.CP_DEFAULT_STYLE|wx.CP_NO_TLW_RESIZE` from the wxPython demo.
Interestingly using only the latter style was enough to fix the problem
but messed up UI otherwise. This combination seems to work OK at least
on Windows XP. Need to test on Windows 7 and Linux.
